src/webgl/uniforms.js

```diff
@@ -130,6 +130,21 @@ function checkUniformValue(value) {
   return ok;
 }
 
+/**
+ * Create the current uniform state object to be sent to the devtools
+ */
+export const getUniformsObject = (program, uniforms) => {
+  if (!program) {
+    return null;
+  }
+
+  return Object.keys(program._uniformSetters).reduce((out, uniformName) => {
```

Comment:
> It's quite disgusting to use for in since you also have to check if it's not an inherited property, but fine

You don't, that linter rule is disabled since it is never an issue in practice.

Also you can do a `for of` on `Object.keys` if that feels better. It is more the `reduce` that looks out of place.